pyanaconda/ui/gui/spokes/datetime_spoke.glade

```diff
@@ -259,6 +260,8 @@
                                       <object class="GtkSwitch" id="networkTimeSwitch">
                                         <property name="visible">True</property>
                                         <property name="can_focus">True</property>
+                                        <property name="valign">center</property>
+                                        <property name="margin_end">6</property>
```

Comment:
Shouldn't we disable the expansion of the widget instead of adding the margins?

Reply:
The margins are in the other direction - horizontal, while the growth is vertical.